.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,105 @@
+name: Bug Report Form
+description: Report an issue related to the Home Assistant Operating System.
+labels: bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Make sure to test with the last version of the Operating System
+        before reporting a bug. If the bug appears to be a regression,
+        make sure to check if the bug indeed disappears from the
+        previous version. Use `ha os update --version x.y` command to
+        downgrade.
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Describe the issue you are experiencing
+      description: Provide a clear and concise description of what the bug is.
+  - type: markdown
+    attributes:
+      value: |
+        ## Environment
+  - type: dropdown
+    validations:
+      required: true
+    attributes:
+      label: What operating system image do you use?
+      options:
+        - generic-x86-64 (Generic UEFI capable x86-64 systems)
+        - khadas-vim3 (Khadas VIM3)
+        - odroid-c2 (Hardkernel ODROID-C2)
+        - odroid-c4 (Hardkernel ODROID-C4)
+        - odroid-n2 (Hardkernel ODROID-N2/N2+)
+        - odroid-xu4 (Hardkernel ODROID-XU4)
+        - ova (for Virtual Machines)
+        - rpi (Raspberry Pi 1)
+        - rpi0-w (Raspberry Pi Zero W)
+        - rpi2 (Raspberry Pi 2)
+        - rpi3 (Raspberry Pi 3 32-bit OS)
+        - rpi3-64 (Raspberry Pi 3 64-bit OS)
+        - rpi4  (Raspberry Pi 4/400 32-bit OS)
+        - rpi4-64  (Raspberry Pi 4/400 64-bit OS)
+        - tinker (ASUS Tinker Board/Tinker Board S)
+      description: >
+        [![Open your Home Assistant instance and show your Home Assistant version information.](https://my.home-assistant.io/badges/info.svg)](https://my.home-assistant.io/redirect/info/)
+
+        `Board` value.
```

Comment:
I would combine the previous and current version:

```suggestion
		Can be found in the [Configuration panel -> Info or Supervisor panel ->
        System tab](https://my.home-assistant.io/badges/info.svg)](https://my.home-assistant.io/redirect/info/). It is listed as the `Board` value.

        [![Open your Home Assistant instance and show your Home Assistant version information.](https://my.home-assistant.io/badges/info.svg)](https://my.home-assistant.io/redirect/info/)
```

Reply:
Yeah I was just thinking about that. For people which don't want/have my setup. Also, My is cool and all, but you need to use middle Mouse/Open in new tab (unfortunately no `target="_blank"` in Markdown) and you then end up with 4 tabs :sweat_smile:

Reply:
Or leave the button out completely